src/wifinode.cpp

```diff
@@ -26,7 +28,7 @@ wifi_node_status_t WifiNode::getStatus() {
 }
 
 void WifiNode::sendFragmentationNeeded(ipv4_headers_t* sourcePacket) {
-  struct raw_pcb* pcb = raw_new(IP_PROTO_ICMP);
+  struct raw_pcb* pcb = raw_new(IP_PROTO_TCP);
```

Comment:
Why did you change this from ICMP to TCP?

Reply:
We already fixed.

Reply:
https://github.com/Instituto-Atlantico/mesh-connect/blob/ivo/transmission80211wifinode/src/wifinode.cpp